x-pack/plugins/security_solution/common/endpoint/service/host_isolation/utils.ts

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export const isVersionSupported = ({
+  currentVersion,
+  minVersionRequired,
+}: {
+  currentVersion: string;
+  minVersionRequired: string;
+}) => {
+  const parsedCurrentVersion = currentVersion.includes('-SNAPSHOT')
+    ? currentVersion.substring(0, currentVersion.length - 9)
```

Comment:
Maybe ok for now, but semver can actually have other suffixes like `-beta`, `-alpha`, `-build`, etc.  ([reference](https://semver.org/#spec-item-9)).
My suggestion here is to instead take everything before the dash:

```suggestion
    ? currentVersion.substring(0, currentVersion.indexOf('-'))
```

Another alternative - use the `semver` package that is already available in kibana's `node_modules`

Reply:
done